firefox_puppeteer/ui/browser/tabbar.py

```diff
@@ -10,9 +10,9 @@
 
 import firefox_puppeteer.errors as errors
 
-from .. import DOMElement
-from ..api.security import Security
-from ..base import UIBaseLib
+from ... import DOMElement
+from ...api.security import Security
+from ...base import UIBaseLib
```

Comment:
Maybe we should start to use absolute imports. The multiple dots will be hard to follow. I suggest `from firefox_puppeteer.api.security import Security`. Similar for the other two. With that we also only need a single block because it can be combined with the errors import.